lib/backpex/live_components/form_component.ex

```diff
@@ -395,12 +395,14 @@ defmodule Backpex.FormComponent do
     end)
   end
 
-  defp return_to_path("continue", _live_resource, _socket, %{current_url: url}, item, :new) do
+  defp return_to_path("continue", live_resource, _socket, %{current_url: url}, item, :new) do
+    primary_value = Map.get(item, live_resource.config(:primary_key))
```

Comment:
Maybe make this public?

```suggestion
    primary_value = live_resource.primary_value(item)
```

Reply:
We had a `primary_value(item, live_resource)` function in `Backpex.HTML.Resource` and a `primary_value(socket, item)` function in `Backpex.LiveResource`.

I removed the function in `Backpex.HTML.Resource` and created a `primary_value(item, live_resource)` function in `Backpex.LiveResource` that us used everywhere.